### PR TITLE
Add staging JH app

### PR DESCRIPTION
**Pull request by 4n4nd**
Related https://github.com/operate-first/support/issues/140

### CODE REVIEW

File: envs/moc/zero/operate-first/jupyterhub-stage.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    namespace: opf-jupyterhub-stage
+  project: operate-first
+  source:
+    path: odh/overlays/moc/zero/jupyterhub-stage
+    repoURL: https://github.com/operate-first/apps.git
+    targetRevision: HEAD
+  syncPolicy:
+    automated:
+      prune: true
+      selfHeal: true
```
Comment (harshad16) [inline envs/moc/zero/operate-first/jupyterhub-stage.yaml:18]:
```suggestion
    automated: {}
```

As this is for stagging namespace, can we keep this just automated for deployment without selfheal or prune?

**Review by harshad16 — COMMENTED** [review]
lgtm,
can we keep this app only automated.

**Review by tumido — COMMENTED** [review]
/lgtm

**Comment by HumairAK** [comment]
did you want to add it to the kustomization?

**Comment by 4n4nd** [comment]
> did you want to add it to the kustomization?

oh lol updated. thanks :+1:

**Review by HumairAK — APPROVED** [review]

**Review by tumido — APPROVED** [review]
/approve

**Pull request merged**